### PR TITLE
 [incubator/taiga] Release 0.2.0 (initial release)

### PR DESCRIPTION
**Pull request by mvitale1989**
**What this PR does / why we need it**:
This PR adds the [taiga](https://taiga.io/) project management application chart into the incubator directory.

**Special notes for your reviewer**:
When provisioning PostgreSQL (`persistence.deployPostgres: true`, which is the default), If using a release name other than `taiga`, the parameter `taiga.dbHost` must be tuned accordingly, as documented in the readme.

**Remarks**:
* This chart is based on [mvitale1989/docker-taiga](https://github.com/mvitale1989/docker-taiga), a fork of the [benhutchins/docker-taiga](https://github.com/benhutchins/docker-taiga) community-mantained image, on which some tunings have been done to adapt it to work well under Kubernetes. There is a [currently open PR](https://github.com/benhutchins/docker-taiga/pull/62) for merging these change back into it, and then switch to using the upstream image.
* The [taiga](https://github.com/taigaio/taiga-back) code itself is released under the AGPL-3.0 license, which grants permission to distribute the software.
* The taiga-events ancillary feature is currently not implemented, but there's no downside to it other than not having live updates in the web interface

### CODE REVIEW

File: incubator/taiga/LICENSE
Comment (mattfarina) [inline incubator/taiga/LICENSE]: This repository is licensed under the Apache 2. Can you please remove the license file.

The license file does not represent the license of the application. Rather, it represents the license of the code within the chart itself.

File: incubator/taiga/requirements.yaml
Comment (mattfarina) [inline incubator/taiga/requirements.yaml]: Since you have used gitignore to block the requirements.lock file you might want to put a range on this requirement. For example, `~0.15.0` which means `>= 0.15.0, < 0.16.0`.

File: incubator/taiga/values.yaml
Comment (mattfarina) [inline incubator/taiga/values.yaml:4]: This image appears to be a new on (for this?) that's manually created. What's the plan for updates and maintenance?

**Review by mattfarina — CHANGES_REQUESTED** [review]

File: incubator/taiga/values.yaml
Comment (mvitale1989) [reply incubator/taiga/values.yaml:4]: That's a fork of [benhutchins/docker-taiga](https://github.com/benhutchins/docker-taiga), which i slightly adapted for clean usage in Kubernetes (added option of using as an init container, and implemented proper termination signal handling). My goal is merging the changes back in the original repository, for which i already have a PR open, and switch back to that image after the merge; but depending on how this process goes i might maintain the fork with these adaptations, and merge upstream changes back into it.

**Comment by quater** [comment]
I have tried this Taiga chart and found it working!

File: incubator/taiga/LICENSE
Comment (mvitale1989) [reply incubator/taiga/LICENSE]: Forgot to confirm here: license deleted

File: incubator/taiga/requirements.yaml
Comment (mvitale1989) [reply incubator/taiga/requirements.yaml]: Done, using the patch range selector now

File: incubator/taiga/values.yaml
Comment (mattfarina) [reply incubator/taiga/values.yaml:4]: It appears https://github.com/benhutchins/docker-taiga doesn't have an up to date public image on docker hub. Once your changes land where will they be shared with others? How will things be kept up to date?

Comment (mvitale1989) [reply incubator/taiga/values.yaml:4]: The author has set an automated build job for the repo on [docker hub](https://hub.docker.com/r/benhutchins/taiga/builds/). The builds failed for a while due to an error that my PR resolves, as you can see in [my automated builds](https://hub.docker.com/r/mvitale1989/docker-taiga/builds/), so once the changes are merged upstream, the latest image will be publicly available.

**Comment by mvitale1989** [comment]
Any news on the review? If not, i'd have some small improvements i'd like to integrate before the that. I'll push them over the weekend.

File: incubator/taiga/values.yaml
Comment (mvitale1989) [reply incubator/taiga/values.yaml:4]: The author of the original image has been unresponsive so far. Given my repo also diverged substantially in the provisioning approach, i will keep using and maintaining my own image.

**Comment by themightychris** [comment]
@mvitale1989 did you give up on benhutchins/docker-taiga#64 or did the needed improvement end up getting merged in another way? Was that the upstream merge this PR got stuck on?

Looking to deploy from this myself soon

**Comment by mvitale1989** [comment]
@themightychris there simply didn't seem to be much interest from neither the upstream image maintainer, nor the helm community for this PR. At least that's my perception, as even after the requested changes there was no feedback from the reviewers.

I am now maintaining [my own fork](https://github.com/mvitale1989/docker-taiga) of the image with all the required fixes and more (readme would need a rewrite though), but i don't think it'd be merged anymore as it has diverged quite a lot.
You can now find the chart in [mvitale1989/helm-taiga](https://github.com/mvitale1989/helm-taiga), which i'm using to deploy my own taiga instances

**Comment by themightychris** [comment]
thanks for keeping your work shared @mvitale1989 ! I've got it bookmarked for my upcoming migration of my Taiga instance to k8s. I'll help you maintain these

**Comment by mvitale1989** [comment]
@themightychris thanks, much appreciated!

**Comment by Zageron** [comment]
What ever happened to this chart? I'd love to see it again.

**Comment by zakkg3** [comment]
Plus 1

**Comment by vkmc** [comment]
+1

**Pull request closed without merging**
